From 41ef72b2b6f6ee4fac6360aa6f9338dfd9579e04 Mon Sep 17 00:00:00 2001
From: Arunpandi Kannan <quic_arunpand@quicinc.com>
Date: Thu, 14 Mar 2024 12:04:33 +0530
Subject: [PATCH] iw: enable offchan scan request command

Enable off-chan scan command by removing the off-channel
module from exclude list in makefile

Signed-off-by: Arunpandi Kannan <quic_arunpand@quicinc.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0c9aff2..993eb59 100644
--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@ _OBJS := $(sort $(patsubst %.c,%.o,$(wildcard *.c)))
 VERSION_OBJS := $(filter-out version.o, $(_OBJS))
 OBJS := $(VERSION_OBJS) version.o
 
-OBJS_FULL = ocb offch cqm wowlan coalesce roc p2p vendor mgmt ap sha256 nan bloom measurements ftm
+OBJS_FULL = ocb cqm wowlan coalesce roc p2p vendor mgmt ap sha256 nan bloom measurements ftm
 ifdef IW_FULL
   CFLAGS += -DIW_FULL
 else
-- 
2.17.1

